storage/transfer_manager/download_chunks_concurrently_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transfermanager
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestDownloadChunksConcurrently(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	bucketName := tc.ProjectID + "-tm-bucket-test"
```

Comment:
It is good to name the test bucket using a test prefix and UUID to prevent collisions. Also, probably worth sharing the bucket across tests if you're going to add more tests for transfermanager. See what I did for the rapid samples including a TestMain https://github.com/GoogleCloudPlatform/golang-samples/blob/main/storage/rapid/rapid_test.go#L1